pkg/providers/vsphere/config/template-md.yaml

```diff
@@ -32,6 +32,18 @@ spec:
 {{- end }}
         nodeRegistration:
           criSocket: /var/run/containerd/containerd.sock
+{{- if .workerNodeGroupTaints }}
+          taints: {{ range .workerNodeGroupTaints}}
```

Comment:
So adding it here is supported now by VSphere?

Reply:
Yes, taints is a part of nodeRegistration.